recipe/bld.bat

```diff
@@ -31,11 +35,16 @@ cmake -S . -B build ^
     -DLLAMA_AVX512=OFF ^
     -DLLAMA_AVX512_VBMI=OFF ^
     -DLLAMA_AVX512_VNNI=OFF ^
-    -DLLAMA_FMA=OFF ^
-    -DLLAMA_F16C=OFF
```

Comment:
Out of curiosity why is the LLAMA_F16 flag removed?

Reply:
`LLAMA_F16C` does not apply to MSVC. It was flagged by CMake.